client/pkg/git/git_test.go

```diff
@@ -7,9 +7,15 @@ import (
 func TestRemoteURL(t *testing.T) {
 	t.Parallel()
 
+	actual := RemoteURL("example.com", "app")
 	expected := "ssh://git@example.com:2222/app.git"
 
-	actual := RemoteURL("example.com", "app")
+	if actual != expected {
+		t.Errorf("Expected %s, Got %s", expected, actual)
+	}
+
+	actual = RemoteURL("deis.10.245.1.3.xip.io:31350", "velcro-underdog")
```

Comment:
It's in there, I only added a new test. The way the diff is displayed makes that a bit hard to see.